pkg/base/linuxutils/partitionhelper/partition_helper.go

```diff
@@ -79,7 +79,7 @@ const (
 	GetPartitionUUIDCmdTmpl = sgdisk + "%s --info=%s"
 
 	// NumberOfRetriesToSyncPartTable how many times to sync fs tab
-	NumberOfRetriesToSyncPartTable = 5
+	NumberOfRetriesToSyncPartTable = 15
```

Comment:
Why we need so many retry times?  Is this expected?

Reply:
This is just the max time we could retry.  why need this,  cause kernel target is still removing,  we need leave time for it to completed removed and then the wipe out action on disk can success.